doc/input-output-reference/src/input-for-output.tex

```diff
@@ -474,51 +474,98 @@ \subsubsection{Field: Name}\label{field-name-1-036}
 
 % table 38
 \begin{longtable}[c]{@{}l@{}}
-\caption{Table of Metered Resource  Types \label{table:table-of-metered-resource-types}} \tabularnewline
-\toprule
-Utility/Fuel Types \tabularnewline
-\midrule
-\endfirsthead
-
-\caption[]{Table of Metered Resource  Types} \tabularnewline
-\toprule
-Utility/Fuel Types \tabularnewline
-\midrule
-\endhead
-
-Electricity \tabularnewline
-NaturalGas \tabularnewline
-Gasoline \tabularnewline
-Diesel \tabularnewline
-Coal \tabularnewline
-FuelOilNo1 \tabularnewline
-FuelOilNo2 \tabularnewline
-Propane \tabularnewline
-OtherFuel1 \tabularnewline
-OtherFuel2 \tabularnewline
-Water \tabularnewline
-DistrictCooling \tabularnewline
-DistrictHeatingWater \tabularnewline
-DistrictHeatingSteam \tabularnewline
-ElectricityPurchased \tabularnewline
-ElectricitySurplusSold \tabularnewline
-ElectricityNet \tabularnewline
-\bottomrule
+	\caption{Table of Metered Resource Types---Fuels \label{table:table-of-metered-resource-types}} \tabularnewline
+	\toprule
+	Utility/Fuel Resource Types \tabularnewline
+	\midrule
+	\endfirsthead
+	
+	\caption[]{Table of Metered Resource Types---Fuels \label{table:table-of-metered-resource-types-Fuels}} \tabularnewline
+	\toprule
+	Utility/Fuel Resource Types \tabularnewline
+	\midrule
+	\endhead
```

Comment:
I feel like this needs to be simplified.  This is a lot of bulk just for a single list of values.  It's not a mandatory change, but would be really nice if cleaned up.

Reply:
@Myoldmopar These tables are not super long---but may not be that short either. When a few of these tables were displayed one after another, there could be a good chance that one or more of them could be displayed in two pages. That's what those two `caption` was used for. It might not be a bad choice to leave these structure for the heading (explained below about the first page's caption vs. the rest pages' caption.

I can change this few if you strongly feel like doing this (or any other thoughts?)---but it could be very valid to leave this structure as is. There could be (many?) more places in the documentation like this as well that used long table.